common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,84 @@
+---
+{
+	"title": "Text-Level Modifiers",
+	"language":"en",
+	"altLangPage":"text-level-modifiers-fr.html",
+	"breadcrumbs":
+		[
+			{
+				"title": "GCWeb home",
+				"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+			}
+		],
+	"secondlevel": false,
+	"dateModified": "2023-07-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
+<dl class="dl-horizontal">
+	<dt><code>.stretched-link</code></dt>
+	<dd>Works in conjunction with position-relative for its stretch limit</dd>
+	<dt><code>.small</code></dt>
+	<dd>Decreases the font size of the element</dd>
+	<dt><code>mark</code></dt>
+	<dd>Highlights the text</dd>
+	<dt><code>.lead</code></dt>
+	<dd>Increases the font size and line height</dd>
+	<dt><code>[class*=text-]</code></dt>
+	<dd>Changes the color of the text</dd>
+	<dt><code>.no-wrap</code></dt>
+	<dd>Prevents the text from wrapping</dd>
+	<dt><code>.fnt-nrml</code></dt>
+	<dd>Resets the font size to the default</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Appearance</h3>
+				<p class="stretched-link">
+					Example 1: Stretched link
+				</p>
+				<p class="small">
+					Example 2: Small text
+				</p>
+				<p class="mark">
+					Example 3: Highlighted text
+				</p>
+				<p class="lead">
+					Example 4: Lead text
+				</p>
+				<p class="text-primary">
+					Example 5: Primary text color
+				</p>
+				<p class="no-wrap">
+					Example 6: No wrap
+				</p>
+				<p class="fnt-nrml">
+					Example 7: Normal font size
+				</p>
+			</div>
+		</div>
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Code</h3>
+				<pre><code>// Example 1:
+					&lt;p <strong>class=&quot;stretched-link&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
```

Comment:
The `<pre>` element do preserve the white spacing. So that example do display with a lot of white spacing which is not well presented.

```suggestion
				<pre><code>// Example 1:
	&lt;p <strong>class=&quot;stretched-link&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
```